src/components/Map/PlantingSiteMap.tsx

```diff
@@ -48,12 +53,12 @@ export default function PlantingSiteMap(props: PlantingSiteMapProps): JSX.Elemen
           boundary: getPolygons(boundary),
           name: `zone-${name}`,
           id: `zone-${id}`,
-          fillColor: 'blue',
-          fillOpacity: 0.5,
+          fillColor: getFillColor(),
```

Comment:
Are planting site, planting zones and plots all white?

Reply:
this is temporary, added a comment in the PR. looking at the designs they all seem white/transparent. i will check with chudo later.